addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/AccountHandler.java

```diff
@@ -467,6 +504,9 @@ private void refreshData() {
                             notificationSounds, musicProviders);
                 }
 
+                // refresh notifications
```

Comment:
Can be removed, I guess ^^ I mean, the method has exact the name of the comment.

Reply:
This is true for this single line, but in this function are multiple code blocks, and for the other the comment is required, so this comment is required to separate it from the other blocks. Anyway, this function should be refactored, I know, but I will do this in another step